jtelegrambotapi-menus/src/main/java/com/jtelegram/api/menu/BackButton.java

```diff
@@ -0,0 +1,28 @@
+package com.jtelegram.api.menu;
+
+import com.jtelegram.api.TelegramBot;
+import com.jtelegram.api.inline.keyboard.InlineKeyboardButton;
+import com.jtelegram.api.user.User;
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+/**
+ * Goes back to the previous menu.
+ */
+public class BackButton implements MenuButton {
+
+    @Nonnull
```

Comment:
Do IDEs such as IntelliJ support this annotation for NPE warnings?

Reply:
Yup! It actually does a great job.